browser/brave_wallet/brave_wallet_provider_impl_unittest.cc

```diff
@@ -131,6 +134,45 @@ class BraveWalletProviderImplUnitTest : public testing::Test {
         ContentSetting::CONTENT_SETTING_ALLOW);
   }
 
+  bool SignMessage(const std::string& address,
+                   const std::string& message,
+                   std::string* signature_in,
```

Comment:
`_out` prefix I think makes more sense for these

Reply:
fixed in 4948f7ce356351ef501571c305ebb9fa57e5c719